### ISSUE TITLE
code wrong

### ISSUE THREAD

**Issue by RuiqingGao** [ZhixiuYe/Intra-Bag-and-Inter-Bag-Attentions#13]
I think your codes has a little mistake. in the train.py.

`            if args.use_RA and args.sent_encoding == "pcnn":
                loss = model.PCNN_ATTRA(total_word, total_pos1, total_pos2,
                                        total_pcnnmask, total_shape, y_batch)
            if args.use_RA and args.sent_encoding == "cnn":
                loss = model.CNN_ATTRA(total_word, total_pos1, total_pos2,
                                       total_pcnnmask, total_shape, y_batch)
            if not args.use_RA and args.sent_encoding == "pcnn":
                loss = model.PCNN_ATT**RA**(total_word, total_pos1, total_pos2,
                                        total_pcnnmask, total_shape, y_batch)
            if not args.use_RA and args.sent_encoding == "cnn":
                loss = model.CNN_ATTBL(total_word, total_pos1, total_pos2,
                                       total_pcnnmask, total_shape, y_batch)`

line 278should be "model.PCNN_ATTBL"
